### PR TITLE
ZER-138 Update ReadMe.md

### PR DESCRIPTION
**Pull request by KostiaFed**
dev
## JIRA

* [ZER-138 JIRA ticket](https://jira.softserve.academy/browse/ZER-138)

## Summary of issue

Create meaningful and detailed documentation for local setup.

Documentation that will include:

How install and configure postgresql.
How install and configure yarn packages.
Explanation of config files for local setup.
Redis
Common system errors fixes (windows, linux etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
...
+## Local setup
+First of all you need rvm to setup project. For the operating system Windows the optimal solution is to use <a href="https://docs.microsoft.com/en-us/windows/wsl/">WSL</a>.
 
 $ `bin/setup`
-or 
-$ `bundle install` 
+or
+$ `bundle install`
+
+<b>PostgreSQL</b>
```
Comment (PrincTwilig) [inline README.md:47]: In the project, the standard password for Postgresql is "password", maybe add information that you need to change the password in the project or set the postgresql password to "password"

photo_2022-07-22_13-10-27

File: README.md [hunk 2]
```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
...
+
+<b>Sidekiq</b>
+
+Simple, efficient background processing for Ruby. You can read more about sidekiq there:
+<a href="https://github.com/mperham/sidekiq">Sidekiq documentation</a>.
+
+Installation:
+$ `bundle add sidekiq`
 
 ## How to run local
```
Comment (PrincTwilig) [inline README.md:119]: In some systems, after restarting them, the postgresql server remains disabled, perhaps at the first start you should enter "sudo service postgresql start"

File: README.md [hunk 3]
```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
...
+<a href="https://classic.yarnpkg.com/lang/en/docs/">yarn documentation</a>.
+
+Installation:
+$ `npm install --global yarn`
+
+<b>Webpacker</b>
+
+To prevent an error when starting the server install webpacker. You can read more about webpacker there:
+<a href="https://guides.rubyonrails.org/webpacker.html">Webpacker documentation</a>.
+
```
Comment (AndriiStegnytskyi) [inline README.md:106]: Please notice about problem with js libraries, and how this command may help:
rm -rf node_modules

**Review by AndriiStegnytskyi — DISMISSED** [review]

**Review by PrincTwilig — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
...
+$ `sudo apt-get install libpq-dev`
+then
+$ `gem install pg`
+
+<b>Database configure</b>
+
+For correct operation of the migration, you need to rename the migration file `20220123171144_create_versions.rb` so that it is processed first.
+
+To create the necessary databases and update them:
+
```
Comment (denysiuk) [inline README.md:68]: Also adding rake db:reset first would help if someone is having troubles with the database.

File: README.md [hunk 5]
```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
...
+<b>pg gem</b>
+
+Under certain circumstances bundle can do not install pg.
+
+To install manually:
+$ `sudo apt-get install libpq-dev`
+then
+$ `gem install pg`
+
```
Comment (denysiuk) [inline README.md:62]: There were a lot of issues with Postgres, which are worth mentioning. Two of the most popular ones:
https://stackoverflow.com/questions/15301826/psql-fatal-role-postgres-does-not-exist
https://stackoverflow.com/questions/55038942/fatal-password-authentication-failed-for-user-postgres-postgresql-11-with-pg

**Review by opazyniuk — CHANGES_REQUESTED** [review]
please rename your branch according to the ID and name of your ticket and apply comments requested above

**Pull request closed without merging**